ui/apps/platform/src/Containers/DeferralConfiguration/VulnerabilitiesConfiguration.tsx

```diff
@@ -65,25 +160,81 @@ function VulnerabilitiesConfiguration() {
                         <Text>Configure deferral behavior for vulnerabilities</Text>
                     </SplitItem>
                     <SplitItem>
-                        <Button variant="primary">Save</Button>
+                        <Button
+                            variant="primary"
+                            isDisabled={!isConfigDirty}
+                            isLoading={isUpdateInProgress}
+                        >
+                            Save
+                        </Button>
                     </SplitItem>
                 </Split>
             </div>
             <Divider component="div" />
             <PageSection variant="light" component="div">
                 <Title headingLevel="h2">Configure deferral times</Title>
-                <Form className="pf-u-py-lg">
-                    <Grid hasGutter>
-                        <NumericSetting value={14} isEnabled />
-                        <NumericSetting value={30} isEnabled />
-                        <NumericSetting value={60} isEnabled />
-                        <NumericSetting value={90} isEnabled />
-                        <BooleanSetting label="Indefinitely" isEnabled />
-                        <BooleanSetting label="Expires when all CVEs fixable" isEnabled />
-                        <BooleanSetting label="Expires when any CVE fixable" isEnabled />
-                        <BooleanSetting label="Allow custom date" isEnabled />
-                    </Grid>
-                </Form>
+                {isConfigLoading && (
+                    <Bullseye>
+                        <Spinner aria-label="Loading current vulnerability deferral configuration" />
+                    </Bullseye>
+                )}
+                {configLoadError && (
+                    <Bullseye>
+                        <EmptyStateTemplate
+                            title="Error loading vulnerability deferral configuration"
+                            headingLevel="h2"
+                            icon={ExclamationCircleIcon}
+                            iconClassName="pf-u-danger-color-100"
+                        >
+                            {getAxiosErrorMessage(configLoadError)}
+                        </EmptyStateTemplate>
+                    </Bullseye>
+                )}
+                {!isConfigLoading && (
```

Comment:
If `configLoadError` has an error to display and `isConfigLoading` is false, I think the layout might look odd, right? The `EmptyStateTemplate` would render and so would the following

Reply:
+1 I'll fix this